pypeit/spectrographs/wht_isis.py

```diff
@@ -171,3 +212,31 @@ def check_frame_type(self, ftype, fitstbl, exprng=None):
         msgs.warn('Cannot determine if frames are of type {0}.'.format(ftype))
         return np.zeros(len(fitstbl), dtype=bool)
 
+    def bpm(self, shape=None, filename=None, det=None, msbias=None, **null_kwargs):
+        """ Generate a BPM
+
+        Parameters
+        ----------
+        shape : tuple, REQUIRED
+        filename : str, REQUIRED for binning
+        det : int, REQUIRED
+        **null_kwargs:
+           Captured and never used
+
+        Returns
+        -------
+        badpix : ndarray
+
+        """
+        # Get the empty bpm: force is always True
+        self.empty_bpm(shape=shape, filename=filename, det=det)
+
+        # Only defined for det=2
```

Comment:
Ah, I see now.  You need a bias image before you can make the `bpm`.
If it is relatively stable (is it?) then you might hard code it.